modules/Bio/EnsEMBL/Analysis/Hive/Config/StarScallopRnaseq.pm

```diff
@@ -0,0 +1,827 @@
+    'assembly_name'       => '',                                                                                      # Name (as it appears in the assembly report file)
+    'registry_file'       => '' || catfile( $self->o('output_path'), "Databases.pm" ),                                # Path to databse registry for LastaZ and Production sync
+    'use_genome_flatfile' => '1',                                                                                     # This will read sequence where possible from a dumped flatfile instead of the core db
+    'skip_rnaseq'         => '0',                                                                                     # Will skip rnaseq analyses if 1
+    'uniprot_version'     => 'uniprot_2019_04',                                                                       # What UniProt data dir to use for various analyses
+    'paired_end_only'     => '1',                                                                                     # Will only use paired-end rnaseq data if 1
+
+    # Keys for custom loading, only set/modify if that's what you're doing
+    'protein_blast_db' => '' || catfile( $self->o('base_blast_db_path'), 'uniprot', $self->o('uniprot_version'), 'PE12_vertebrata' ),    # Blast database for comparing the final models to.
+    'protein_blast_index' => '' || catdir( $self->o('base_blast_db_path'), 'uniprot', $self->o('uniprot_version'), 'PE12_vertebrata_index' ),    # Indicate Index for the blast database.
+    'protein_entry_loc' => catfile( $self->o('base_blast_db_path'), 'uniprot', $self->o('uniprot_version'), 'entry_loc' ),                       # Used by genscan blasts and optimise daf/paf. Don't change unless you know what you're doing
+
+########################
+# Pipe and ref db info
+########################
+
+    'pipe_db_name' => $self->o('dbowner') . '_' . $self->o('production_name') . '_pipe_' . $self->o('release_number'),
+    'dna_db_name'  => $self->o('dbowner') . '_' . $self->o('production_name') . '_core_' . $self->o('release_number'),
+
+    'reference_db_name'   => $self->o('dna_db_name'),
+    'reference_db_server' => $self->o('dna_db_server'),
+    'reference_db_port'   => $self->o('dna_db_port'),
+
+    'star_rnaseq_for_layer_db_server' => $self->o('databases_server'),
+    'star_rnaseq_for_layer_db_port'   => $self->o('databases_port'),
+
+    'scallop_initial_db_server' => $self->o('databases_server'),
+    'scallop_initial_db_port'   => $self->o('databases_port'),
+
+    'scallop_blast_db_server' => $self->o('databases_server'),
+    'scallop_blast_db_port'   => $self->o('databases_port'),
+
+    # This is used for the ensembl_production and the ncbi_taxonomy databases
+    'ensembl_release' => $ENV{ENSEMBL_RELEASE},    # this is the current release version on staging to be able to get the correct database
+
+    databases_to_delete => ['reference_db'],
```

Comment:
You only want to have databases created in this pipeline in the array. So definitely not reference_db nor dna_db but all the others

Reply:
Removed reference_db and added all the other databases.